src/react/index.ts

```diff
@@ -20,14 +20,17 @@ export function useAutoAnimate<T extends Element>(
     if (node instanceof HTMLElement) {
       setController(autoAnimate(node, options))
     } else {
-      setController(undefined);
+      setController(undefined)
     }
-  }, [])
-  const setEnabled = (enabled: boolean) => {
-    if (controller) {
-      enabled ? controller.enable() : controller.disable()
-    }
-  }
+  }, [options])
```

Comment:
This doesn't seem correct - if the `options` object is an object literal, then this will cause the `element` callback to be re-evaluated on every render